tests/README.md

```diff
@@ -20,32 +25,100 @@ Go to the `tests` folder.
 - Run sanity tests to check the host setup:
 
 ```
-$ sudo ./run.sh pytest pytest/test_host_*.py
+$ sudo tox -e test_host
```

Comment:
I needed to pass -E to sudo to get this to work, with `TDXTEST_GUEST_IMG` exported to my environment. I think this should be added to the README for consistency with the checkbox section.

Reply:
Good idea, just added this